Ginger/GingerCore/Actions/ActSikuli.cs

```diff
@@ -164,18 +164,16 @@ public string ProcessNameForSikuliOperation
             }
         }
 
-        private string mPatternSimilarity = "70";
-
         [IsSerializedForLocalRepository]
         public string PatternSimilarity
         {
             get
             {
-                return mPatternSimilarity;
+                return GetOrCreateInputParam(nameof(PatternSimilarity)).Value;
```

Comment:
if storing field on input params then input params are by serialized from Act.cs inputs values list. 
So IsSerializedForLocalRepository is not needed for this field.